python_scripts/03_categorical_pipeline_column_transformer.py

```diff
@@ -220,7 +220,7 @@
 # section we will use such a model called **gradient-boosting trees** and
 # evaluate its statistical performance. More precisely, the scikit-learn model
 # we will use is called `HistGradientBoostingClassifier`. Note that boosting
-# models will be covered in more details in a future module.
+# models will be covered in more detail in a future module.
```

Comment:
Good catch it seems like "in more detail" is the correct wording e.g. https://ell.stackexchange.com/questions/93460/in-more-details-or-in-detail!